app/Http/Controllers/EmailWebhookController.php

```
@@ -0,0 +1,57 @@
+                'message' => $success ? 'Email processed successfully' : 'Failed to process email'
+            ]);
+        } catch (\Exception $e) {
+            Log::error('Email webhook error', [
+                'error' => $e->getMessage(),
+                'request_data' => $request->all(),
+            ]);
+            return response()->json(['error' => 'Internal server error'], 500);
+        }
+    }
+
+    private function verifyWebhookSignature(Request $request): bool
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)